<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MEX_AdminPanel</class>
 <widget class="QWidget" name="MEX_AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MEX - Admin Panel</string>
  </property>
  <widget class="QLineEdit" name="edtNewPW">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="lwUsers">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowPW">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Hash</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteUser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowUsers">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangePW">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtProductName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="lwProducts">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowProducts">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Products</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddProduct">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteProduct">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Product</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtProductSymbol">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxIndices">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblIndex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Index</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSymbol">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Symbol</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lwUsers</tabstop>
  <tabstop>btnShowUsers</tabstop>
  <tabstop>btnShowPW</tabstop>
  <tabstop>btnDeleteUser</tabstop>
  <tabstop>edtNewPW</tabstop>
  <tabstop>btnChangePW</tabstop>
  <tabstop>lwProducts</tabstop>
  <tabstop>btnShowProducts</tabstop>
  <tabstop>btnDeleteProduct</tabstop>
  <tabstop>cBoxIndices</tabstop>
  <tabstop>edtProductSymbol</tabstop>
  <tabstop>edtProductName</tabstop>
  <tabstop>btnAddProduct</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
